myRental/logs/Error.log

```diff
@@ -2,3 +2,12 @@ Fri Feb 09 2018 20:36:42 GMT-0600 (CST) - LOG::: | ERROR ᑀ Test
 Thu Feb 15 2018 19:51:30 GMT-0600 (CST) - LOG::: | ERROR ᑀ {}
 Thu Feb 15 2018 20:16:18 GMT-0600 (CST) - LOG::: | ERROR ᑀ {}
 Thu Feb 15 2018 20:16:31 GMT-0600 (CST) - LOG::: | ERROR ᑀ {}
+Sun Feb 18 2018 14:22:15 GMT-0600 (CST) - LOG::: | ERROR ᑀ {}
```

Comment:
Since we just have to make sure this log file exists, we don't need to be committing changes to it